File: doc/setup/mysql_connect/mysql_connect.m

driverClassName = 'com.mysql.jdbc.Driver';
classLoader = com.mathworks.jmi.ClassLoaderManager.getClassLoaderManager;
%driverClass = classLoader.loadClass(driverClassName);
classLoader.loadClass(driverClassName);

% jdbc:mysql://[host][,failoverhost...]
% [:port]/[database]
% [?propertyName1][=propertyValue1]
% [&propertyName2][=propertyValue2]

connStr = 'jdbc:mysql://ndsw-cdcu-yello:3306/dbCDCUtest';
con = java.sql.DriverManager.getConnection(connStr,'adminCDCU','schmee#schm44');

query = 'SELECT name from dumbTable';

stmt = con.createStatement();
rs = stmt.executeQuery(query);
if ~rs.next()
    fprintf(1, '\nError, bad database connection\n'); stmt.close(); return;
end

name = char(rs.getString('name'));
stmt.close();
display(sprintf('query "%s" returned %s',query,name));
con.close();

File: doc/setup/mysql_connect/mysql_connect.py

import mysql.connector

cnx = mysql.connector.connect(user='adminCDCU', password='schmee#schm44',
                              host='ndsw-cdcu-yello',
                              database='dbCDCUtest')

cursor = cnx.cursor()

query = ('SELECT name from dumbTable')

cursor.execute(query)

for (name,) in cursor:
  print('Name is ' + name)

cursor.close()
cnx.close()

